Android/Fundamentals/android-basics/android-files-generated.md

```diff
@@ -0,0 +1,115 @@
+# android-files-generated
```

Comment:
We talked about showing the folder and talking over the roles of the different files, rather than showing the file itself (since we don't care if someone can read this file, only that they know what it's for)

Reply:
I updated to show the roles of the different roles than showing the file itself. Much agreed.